pkg/server/server.go

```diff
@@ -86,7 +86,7 @@ func (s *Server) HandleCNIRequest(cmd string, k8sArgs *types.K8sArgs, cniCmdArgs
 
 // HandleDelegateRequest is the CNI server handler function; it is invoked whenever
 // a CNI request is processed as delegate CNI request.
-func (s *Server) HandleDelegateRequest(cmd string, k8sArgs *types.K8sArgs, cniCmdArgs *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) ([]byte, error) {
+func (s *Server) HandleDelegateRequest(cmd string, k8sArgs *types.K8sArgs, cniCmdArgs *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo, annotation *api.DelegateAnnotation) ([]byte, error) {
```

Comment:
nit: I would name the attribute something other than `annotation`, which is too generic - it can be pretty much anything.

I would prefer something more descriptive - not sure `interfaceAttributes` would be a good fit, because this also has the `cni args`. 

But, since I can't provide an accurate suggestion, I guess yours is good enough.

Reply:
changed.